src/components/util/validation-functions.js

```diff
@@ -98,6 +98,21 @@ export function validateField(value, toValidate) {
         }
     }
 
+    if (toValidate.isValueGreaterOrEqualThan !== undefined) {
+        if (value && toValidate.isValueGreaterOrEqualThan) {
+            // TODO: remove replace when parsing behaviour will be made according to locale
```

Comment:
?

Reply:
same comment for the other code lines already present in this function ...